src/classes.js

```diff
@@ -1,5 +1,60 @@
 class Foo {
+  constructor(bar) {
+    this.bar = bar;
+  }
+}
+
+var bar = new Foo('bar')
 
+class Dog {
+  constructor(says) {
```

Comment:
`Dog`s will always say `life is ruff` so you don't need to take any arguments here.